packit_service/worker/testing_farm.py

```diff
@@ -118,6 +118,17 @@ def run_testing_farm_on_all(self):
         )
 
     def run_testing_farm(self, chroot: str) -> TaskResults:
+        # inform users about temporarily disabled testing farm
+        self.report_status_to_test_for_chroot(
+            state=CommitStatus.success,
```

Comment:
I hope that `success` will not be misleading to users. But I agree that only success will not break the whole PR status.

Reply:
I thought there is also neutral but it's not:
https://github.com/packit/packit-service/issues/760
https://docs.github.com/en/rest/reference/repos#create-a-commit-status

yes, either set error, success or don't set it at all

Reply:
Yes, we can't use neutral for now. (Would be nice to mention in the linked issue.)

> don't set it at all
This might require more changes. (And we don't have place to let user know.)

Reply:
> > don't set it at all
> This might require more changes. (And we don't have place to let user know.)

agreed, I'd say all three are "bad" and not setting at all is probably the worst solution

Reply:
yes, I also wanted to set neutral but that's not possible, so I will leave there `success`